src/imageview.cpp

```diff
@@ -317,5 +346,26 @@ QRect ImageView::sceneToViewport(const QRectF& rect) {
   return QRect(topLeft, bottomRight);
 }
 
+void ImageView::blankCursor() {
+    viewport()->setCursor(Qt::BlankCursor);
+}
+
+void ImageView::hideCursor(bool enable) {
+  if(enable) {
+    if(cursorTimer_) delete cursorTimer_;
+    cursorTimer_ = new QTimer(this);
```

Comment:
Wouldn't it be better to just create/initialize the `cursorTimer_` object only in case not existing, t.m.:
```c++
if (!cursorTimer_)
{
  cursorTimer_ = new QTimer(this);
...
```

...and second question: Does a `QTimer` object use much resources -> why not to create/initialize it just once in the `ImageView` constructor and use it later only when needed?